skimage/filters/thresholding.py

```diff
@@ -366,3 +368,89 @@ def threshold_li(image):
             new_thresh = temp + tolerance
 
     return threshold + immin
+
+def threshold_minimum(image, nbins=256, bias='min'):
+    """Return threshold value based on minimum method.  A histogram is computed
```

Comment:
Use a one line short description, 1 empty line and then a little more elaborate description using a few sentences. Sorry, but this is a convention in skimage. To make it easy you could just split after the first sentence.